samples/csharp/getting-started/DeepLearning_TensorFlowEstimator/ImageClassification.Predict/Program.cs

```diff
@@ -8,9 +8,10 @@ namespace ImageClassification.Predict
 {
     class Program
     {
-        static async Task Main(string[] args)
+        static void Main(string[] args)
```

Comment:
Removed **Async** keyword as there is no use of calling any method asynchronously.